File: srm161/card_count.py
class CardCount:
	def dealHands(self, numPlayers, deck):
		cards = ['' for i in range(numPlayers)]
		n = len(deck) - len(deck) % numPlayers
		new_deck = deck[:n]
		for i in range(len(new_deck)):
			cards[i%numPlayers] += new_deck[i]
		
		return cards

File: srm161/cardCount.java
public class CardCount {
	public String[] dealHands(int numPlayers, String deck) {
		String[] hands = new String[numPlayers];
		int n = deck.length() - deck.length() % numPlayers;
		String newDeck = deck.substring(0, n);
		for (int i = 0; i < hands.length; i++) {
			hands[i] = "";
			}
		for (int i = 0; i < newDeck.length(); i++) {
			hands[i % numPlayers] += newDeck.charAt(i);
			}
		return hands;
		}
	} 